src/store/status/getters.ts

```diff
@@ -21,4 +24,6 @@ export const getters: GetterTree<TxStatus, RootState> = {
     const status = state.txDetails ? state.txDetails.status : '';
     return setStatusMessage(state.type, status);
   },
+  [constants.STATUS_GET_RELEASE_TIME_TEXT]: (state: TxStatus):
+    string => state.estimatedReleaseTimeInMinutes.humanize(true),
```

Comment:
```suggestion
    string => state.estimatedReleaseTimeInMinutes.humanize(true, { h: 34 }),
```
I think we can add a threshold at least to show time in hours instead of days.